tests/core/Operation.spec.ts

```diff
@@ -187,35 +214,120 @@ describe('Operation', async () => {
       const expectedError = new SidetreeError(ErrorCode.OperationUpdatePatchServiceEndpointNotString);
       expect(() => { Operation.validateUpdatePayload(updatePayload); }).toThrow(expectedError);
     });
+  });
 
-    describe('applyPatchesToDidDocument', async () => {
-      it('should prevent the same id for multiple keys', async () => {
-        const didDocument: DocumentModel = {
-          '@context': 'someContext',
-          id: 'someId',
-          publicKey: [{ id: 'aRepeatingId', type: 'someType', usage: 'some usage', controller: 'someId' }],
-          service: []
-        };
-        const patches = [
-          {
-            action: 'add-public-keys',
-            publicKeys: [
-              { id: 'aRepeatingId', type: 'thisShouldNotShowUp', usage: 'thisShouldNotShowUp' },
-              { id: 'aNonRepeatingId', type: 'someType', usage: 'some usage' }
-            ]
-          }
-        ];
+  describe('applyPatchesToDidDocument()', async () => {
+    it('should prevent the same id for multiple keys', async () => {
+      const didDocument: DocumentModel = {
+        '@context': 'someContext',
```

Comment:
context is not valid, should be a uri or an object.

Latest W3C Context: "https://www.w3.org/ns/did/v1" will work.

Reply:
Note any fields defined but not present in the context above will cause errors when processing the DID Document as JSON-LD...

We have defined: https://docs.element-did.com/contexts/sidetree/sidetree-v0.0.jsonld

for "usage", but not updated our context to reflect this yet.

The most correct thing for us to do would be:

"@context": ["https://www.w3.org/ns/did/v1", "https://identity.foundation/contexts/sidetree/sidetree-v0.0.jsonld"]

This is not currently possible, but I want to put it on the radar.

Reply:
Changed to `https://www.w3.org/ns/did/v1`